slides/src/content/unit-testing.md

```diff
@@ -115,9 +126,11 @@ describe('CapitalizePipe', () => {
 <!-- .slide: id="unit-testing-add-a-test" -->
 ## Add a Test
 
+- Use `fdescribe` to tell Jasmine to "**f**ocus" on this test -- we have several broken tests that we'll ignore for now.
```

Comment:
1.  Maybe turn the second half of the sentence (after `--`) into a sub-point?
2.  Do I have to edit the test file whenever I want to focus on particular tests, or can I specify tests from the command line when I run things?